src/hooks/useWalletSectionsData.ts

```diff
@@ -41,6 +39,11 @@ export default function useWalletSectionsData({
 
   const { isCoinListEdited } = useCoinListEdited();
 
+  const uniqueImageTokens = useMemo(
+    () => allUniqueTokens.filter(uniqueToken => !uniqueToken.animation_url),
```

Comment:
Can we extract the condition here to something meaningful like:

`isImage = (uniqueToken) => !uniqueToken.animation_url`